### DISCUSSION TITLE
Circuit box loses values/clamp/etc when game is saved while circuit box is in inventory

### DISCUSSION THREAD

**Discussion by gerakis** [FakeFishGames/Barotrauma#14370] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If you make a circuit in a circuit box, it does not save values inside the components if the circuitbox is inside a player's inventory when the game is saved and then loaded. Meaning any values saved to a memory component are lost, the clamp max and clamp min on divide components, etc.

My guess is this is something related to the load, not the save, because when I looked at my characters XML file when he was saved carrying the circuit box, it had the values saved correctly to each component.

### Reproduction steps

Start a multiplayer campaign.
Get a circuit box
Place a memory component inside
Place a divide component inside
Set the memory "value" slot on the memory component to some number
Set the clamp max and clamp min on the divide component to some number
Exit the circuit box and remove it from the wall with a wrench
With the circuit box in your inventory, save the game
Load the game
Put the box back on the wall
Notice all the values have been reset, as though you never changed them to begin with

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, fix coming up in a future update!
